examples/with-compiled-css/pages/class-names-box.js

```diff
@@ -0,0 +1,18 @@
+import { ClassNames } from '@compiled/react'
```

Comment:
In Next.js, the files that are placed in the `/pages` folder become routes
A general practice is to put any included components in a separate root folder, like `/components`, so they don't become routes in the app

Only `index.js` should stay in `/pages` folder, as far as I can see, the others are only components